### PR TITLE
HP-11: Add Bulma.io to project and Use Django-Scss-Processor

### PR DESCRIPTION
**Pull request by Audiosutras**
Clarifying what James said on the call, when he said 8-point grid he meant 8 px spacing. So, the spacing between items will be `4px` / `8px` / `16px` / `32px` in the wireframes. This also converts to `0.25rem` / `0.5rem` / `1rem` / `2rem` respectively.

This PR:
1) Links the `init.scss` stylesheet to the `base.html` template
2) Imports [bulma.io](https://bulma.io/), a css only framework, into the `init.scss` stylesheet
3) Imports the base.scss stylesheet into the `init.scss` stylesheet.

Styling will be expanded upon in HP-9. To see basic styles being imported from the bulma.io navigate to `/home`.

### CODE REVIEW

File: hip/static/styles/init.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* Initialization File
+
+This file will manage all scss file imports.
+*/
+
+// ------- bulma.io css framework ----------------- //
+
+@import "https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css";
```
Comment (dchukhin) [inline hip/static/styles/init.scss:8]: Minor note: it looks like version 0.9.2 has been released

**Review by dchukhin — APPROVED** [review]
Looks like a good start to the base template

File: hip/templates/base.html [hunk 2]
```diff
@@ -33,20 +35,21 @@
     {# favicon #}
     <link rel="shortcut icon" href="{% static 'favicon.ico' %}" type="image/x-icon">
 
-    {# Base stylesheets #}
-    <link href="JT_BASE_STYLESHEET" rel="stylesheet" type="text/css" />
-
-
+    {# sass Stylesheets #}
+    <link href="{% sass_src 'styles/init.scss' %}" rel="stylesheet" type="text/css" />
```
[the remark this answers is not in the document]
Comment (Audiosutras) [reply hip/templates/base.html:39]: No, it's a convention I just gave it. Based on the `django-sass-processor` section of the philly-hip docs it looks like `base.scss` should be considered a page that corresponds to `base.html`. I will rename `init.scss` to `bundle.scss`.

[the remark this answers is not in the document]
Comment (Audiosutras) [reply hip/templates/base.html:39]: The only thing about renaming this `bundle.scss` is (at least initially) we will have to remember that this file is where we will import all other scss (page) files.

[the remark this answers is not in the document]
Comment (dchukhin) [reply hip/templates/base.html:39]: I wouldn't say I feel strongly about it, so feel free to do whichever seems better for you

[the remark this answers is not in the document]
Comment (Audiosutras) [reply hip/templates/base.html]: changed to `bundle.scss`. done

File: hip/static/styles/init.scss (hunk 1 above)
Comment (Audiosutras) [reply hip/static/styles/init.scss]: will update

Comment (Audiosutras) [reply hip/static/styles/init.scss]: done

File: apps/hip/templates/home.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block title %}
+  Philadelphia Health Information Portal
```
[the remark this answers is not in the document]
Comment (Audiosutras) [reply apps/hip/templates/home.html:4]: done

File: apps/hip/templates/home.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+{% endblock %}
+
+{% block content %}
+  <div>
+    <h3>Welcome to the homepage for PDHP</h3>
```
[the remark this answers is not in the document]
Comment (Audiosutras) [reply apps/hip/templates/home.html:9]: done

**Pull request merged**